src/main/java/org/sagebionetworks/bridge/spring/controllers/MasterSchedulerController.java

```diff
@@ -0,0 +1,96 @@
+package org.sagebionetworks.bridge.spring.controllers;
+
+import java.util.List;
+
+import static org.sagebionetworks.bridge.Roles.ADMIN;
+import org.sagebionetworks.bridge.models.ResourceList;
+import org.sagebionetworks.bridge.models.StatusMessage;
+import org.sagebionetworks.bridge.models.TimestampHolder;
+import org.sagebionetworks.bridge.models.schedules.MasterSchedulerConfig;
+import org.sagebionetworks.bridge.services.MasterSchedulerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin
+@RestController
+public class MasterSchedulerController extends BaseController {
+    static final StatusMessage DELETED_MSG = new StatusMessage("Scheduler config deleted.");
+
+    
+    @Autowired
+    MasterSchedulerService schedulerService;
+
+    public void setMasterSchedulerService(MasterSchedulerService schedulerService) {
```

Comment:
We usually mark these setters as final. It's unlikely to come up again, but if you subclass a class and override the setter, the parent class's setter will never be called, and then bad things happen.